packages/cli/src/commands/generate/service/__tests__/scenario.test.js

```diff
@@ -85,6 +85,13 @@ describe('the scenario generator', () => {
     expect(value).toEqual(parseFloat(value))
   })
 
+  test('scenarioFieldValue returns a float for Float types', () => {
+    const field = { type: 'Float' }
+    const value = service.scenarioFieldValue(field)
+
```

Comment:
I don't think this test does much lol, I appreciate its just a copy paste from the above.

I think maybe add a line (pseudocode)-
```
expect(typeof value).toBe(number)
```

Reply:
It sets the field to

`const field = { type: 'Float' }`

and the one above is a Decimal.

And the both expect a Float.

` expect(value).toEqual(parseFloat(value))`

Reply:
I'll see if there is a way to check if a proper Float value rather than comparing the two.

Reply:
Sweet!

Reply:
Oh... I just re-read the code, actually think this is fine DT! Resolving